eng/pipelines/templates/steps/analyze.yml

```diff
@@ -119,4 +119,8 @@ steps:
       TestMarkArgument: ${{ parameters.TestMarkArgument }}
       AdditionalTestArgs: ${{parameters.AdditionalTestArgs}}
 
+  - template: /eng/common/pipelines/templates/steps/create-apireview.yml
```

Comment:
Do you know if this will be able to find the correct path based one where they were downloaded?

Reply:
It's not working properly right now. I'm going to be updating it to download the artifact, then passing that artifact directory to it.

Reply:
Passes tests, but it's wrong.